pandas/tests/reshape/test_merge.py

```diff
@@ -1344,32 +1341,194 @@ def f():
                             .09604978, -.06524096, .03532373,
                             .03025441, .036997, None, None]
             ))
-            .set_index(["household_id", "asset_id", "t"]))
+            .set_index(["household_id", "asset_id", "t"])
+            .reindex(columns=['share', 'log_return']))
+
+        result = (merge(household.reset_index(), log_return.reset_index(),
+                  on=['asset_id'], how='outer')
+                  .set_index(['household_id', 'asset_id', 't']))
+
+        assert_frame_equal(result, expected)
+
+
+class TestJoinMultiMulti(object):
+    @pytest.fixture
```

Comment:
I would simply put the fixture outside the class, right above TestJoinMultiMulti, then you can just pass them in as needed